internal/aws/metrics/metric_calculator_test.go

```diff
@@ -130,14 +139,14 @@ func TestMapWithExpiryCleanup(t *testing.T) {
 	assert.Equal(t, 1, store.Size())
 	store.Unlock()
 
-	time.Sleep(time.Second + time.Millisecond)
+	time.Sleep(time.Second * 2)
```

Comment:
This is a significant increase. Why is this needed?

Reply:
There's actually no need. I reduced it even further by dropping the TTL and the sleep time together.